src/components/network/equipment-table.js

```diff
@@ -6,11 +6,18 @@ import {
     HEADER_ROW_HEIGHT,
     ROW_HEIGHT,
 } from './config-tables';
+import { useTheme } from '@mui/material/styles';
 
 export const EquipmentTable = (props) => {
     const gridRef = useRef();
 
     const [fixedColumnsCount, setFixedColumnsCount] = useState(0);
+    const theme = useTheme();
+    const [scrollTopIndex, setScrollTopIndex] = useState(
```

Comment:
Maybe rename it to something like `scrollTopLock` since its usage is more about locking/unlocking the scroll and the value of the index doesn't seem to be consumed